docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '3.2'
+services:
+  mariadb:
+    image: mariadb:10.2.8
```

Comment:
Could we use `mariadb:10.3` here?

Reply:
`10.2.8` is flagged as `latest`, thats why i chose it.  Version `10.3` might work as well.

Reply:
Maybe even `10`? (The on-disk database format should only change between major releases.)